server/test/services/DateConverterTest.java

```diff
@@ -38,7 +38,7 @@ public void parseIso8601DateToLocalDate_isSuccessful() {
 
   @Test
   public void renderDate_isSuccessful() {
-    String expectedResult = "2020/01/01";
+    String expectedResult = "2020-01-01";
```

Comment:
Will this change cause other date renderings to change in the UI? If so, are we sure that dashes are what we want rather than slashes in those cases?

I'd like to better understand why this is necessary, especially since all of the screenshots I see use "/".

Reply:
https://stackoverflow.com/questions/14212527/how-to-set-default-value-to-the-inputtype-date -- I knew you would ask this :).. also i checked the code-base, this is not used anywhere, hence could safely change it.

Reply:
It is taking a LocalDate...

Reply:
Correct. I'm suggest naming the method `formatIso8601Date` with a param of `LocalDate`.